services/release/release.go

```diff
@@ -69,7 +69,7 @@ func CreateRelease(gitRepo *git.Repository, rel *models.Release, attachmentUUIDs
 	if err = createTag(gitRepo, rel); err != nil {
 		return err
 	}
-
+	notification.NotifyNewReleaseTag(gitRepo, rel)
```

Comment:
Notifications should be triggered after `models.InsertRelease(rel)`?

Reply:
@lunny not really as tag is already created in repository, even if model save fails webhook should still fire as it would happen if you push tag from git cli

Reply:
But on `NotifyNewReleaseTag` it may get release's ID which is not exist.

Reply:
Imho it does not use release id